Library/Homebrew/test/formula_installer_spec.rb

```diff
@@ -87,10 +87,8 @@ def temporary_install(formula)
   end
 
   specify "Formula is not poured from bottle when compiler specified" do
-    expect(ARGV.cc).to be nil
-
-    cc_arg = "--cc=clang"
-    ARGV << cc_arg
+    expect(Homebrew.args.cc).to be nil
+    allow(Homebrew.args).to receive(:cc).and_return("clang")
```

Comment:
Just wanna make sure: This is a valid replacement in tests for adding values to `ARGV`, right?

Reply:
Yep 👍